racksdb/web/ui/src/views/DatacenterRoomView.vue

```diff
@@ -89,14 +126,15 @@ const props = defineProps({
 <template>
   <BreadCrumbs :datacenterName="props.datacenterName" :datacenterRoom="props.datacenterRoom" />
 
-  <SearchBar
-    v-if="datacenters.length"
-    viewTitle="Datacenter Room"
-    searchedItem="datacenter"
-    :items="datacenters"
-  />
-
-  <h2 class="flex justify-center py-10 capitalize text-3xl">{{ datacenterRoom }} room</h2>
+  <div class="flex justify-end items-center p-4 px-20">
+    <input
+      type="text"
+      v-model="input"
+      class="w-96 rounded-md focus:border-purple-700 focus:outline-none border-2 border-solid bg-white py-1.5 pl-3 pr-10 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 sm:text-sm sm:leading-6"
+      :placeholder="`Filter rack by name`"
```

Comment:
Why do you double quote a simple static string here?

Reply:
It's a mistake, I reuse the code from another view. In the other view, I tried to use template literals without success and I forgot to erase them.